### ISSUE TITLE
Disable mini map

### ISSUE THREAD

**Issue by voidkahn** [moagrius/TileView#162]
At this moment it seems the mini map can't be removed and this is a huge issue when it covers the areas where your markers are on.

Is there any solution to this?

**Comment by moagrius** [comment]
I'm not sure what you mean - I have no notion of a "mini-map" - are you sure you're posting in the right project?  Perhaps this library is being included in another project?

**Comment by voidkahn** [comment]
Yes this is the correct project. I guess this does count as a bug then and not a feature.

I have always had a mini map even when using the default example with the "house" map.

You can see I have 2 markers, an excavator and a hauler. The mini map on top is always there.
Maybe I think the problem is that the first tile (since this minimap zooms like other tiles, gets bigger when you zoom in) is an image of the whole map instead of the actual first tile.

![screen2](https://cloud.githubusercontent.com/assets/11520945/6705020/d73cf10a-cd50-11e4-9efe-3eff1e4fd176.png)
![screen](https://cloud.githubusercontent.com/assets/11520945/6705031/f3e8be7e-cd50-11e4-9fe0-f5b5ff65f927.png)

Relevant code:
tileView = new TileView(mActivity);
tileView.setSize(WIDTH, HEIGHT);

tileView.addDetailLevel(1f, "tiles/1000_%col%_%row%.png", "downsamples/map.png");
tileView.addDetailLevel(0.5f, "tiles/500_%col%_%row%.png", "downsamples/map.png");
tileView.addDetailLevel(0.25f, "tiles/250_%col%_%row%.png", "downsamples/map.png");
tileView.addDetailLevel(0.125f, "tiles/125_%col%_%row%.png", "downsamples/map.png");

tileView.defineRelativeBounds(LEFT, TOP, RIGHT, BOTTOM);

// Set the default zoom (zoom out by 4 => 1/4 = 0.25)
tileView.setScale(0.25);

**Comment by voidkahn** [comment]
I think I may know what causes this bug. WIDTH and HEIGHT are 4 times bigger than the original image.

I had to do this because this made it possible to "zoom in more".

But by doing this apparently I get two tileviews. One big that I use and where the markers are placed and one small at the top which matches what I get if I set width and height correctly.

Still, I find this behaviour extrange. I guess the solution will be to make the image bigger with something simple as paint and then redo the tiles and set correct Width and Height values.

At least this gives you and Idea of a simple mini map feature, simply switch the top tiles with the downsampled whole image.

**Comment by voidkahn** [comment]
I have made the original image bigger and obtained new tiles, still getting the same problem.

I'll keep looking into it and see if I can fix it by changing some of the code. I'll report back later.

**Comment by voidkahn** [comment]
Update:

I probably made some mistake before but now I have made the original image bigger 4 times. I created the tiles and set the width and height values correctly to match the now bigger image.

This way I don't get a "mini map".

However, I also noticed that I shouldn't be needing this, the TileView should allow me to zoom in but it didn't. It barely zoomed in, not even to double the size.

The tileview was inside a LinearLayout with width 900 and height 600. The tileview was of width 6000 and height 3600 but the actual size of the image was 4 times smaller. This gave me a mini map (always when using a bigger size for the tileview than the actual image, no matter what image).
For some reason when the tileview had the real size I could barely zoom in.

Now that the image is 6000 x 3600 I can zoom in without issue and no "mini map".
